superset/charts/schemas.py

```diff
@@ -873,18 +873,20 @@ class Meta:  # pylint: disable=too-few-public-methods
     )
     applied_time_extras = fields.Dict(
         description="A mapping of temporal extras that have been applied to the query",
-        required=False,
+        allow_none=True,
         example={"__time_range": "1 year ago : now"},
     )
-    filters = fields.List(fields.Nested(ChartDataFilterSchema), required=False)
+    filters = (fields.List(fields.Nested(ChartDataFilterSchema), allow_none=True),)
```

Comment:
Is this supposed to be a tuple?

Reply:
It is not. Accidentally ddded an extra comma then Pylint autoformatting made it a tuple.

Thanks for catching it!